### ISSUE TITLE
pip install fails on debian 8

### ISSUE THREAD

**Issue by pritstift** [makinacorpus/django-geojson#83]
When installing django-geojson with pip (python 3.5) the installation fails.

```
Collecting django-geojson==2.9.1 (from -r requirements.txt (line 17))
  Using cached django-geojson-2.9.1.zip
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-zi6lydcz/django-geojson/setup.py", line 15, in <module>
        open(os.path.join(here, 'CHANGES')).read(),
      File "/webapps/datahub-dev/lib/python3.5/encodings/ascii.py", line 26, in decode
        return codecs.ascii_decode(input, self.errors)[0]
    UnicodeDecodeError: 'ascii' codec can't decode byte 0xc3 in position 2118: ordinal not in range(128)
```
I read somewhere that this could be due to a memory problem. However, created a swapfile and it still doesn't work.

**Comment by Gagaro** [comment]
Looks similar to #59

**Comment by shayneoAtNorwood** [comment]
I'm getting this on OSX Sierra  in a virtualenv with django 1.11 on py 3.6

**Comment by Gagaro** [comment]
With the 2.10.0?

Could you paste the entire trace?

Thanks

**Comment by 7wonders** [comment]
```
Downloading django-geojson-2.10.0.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-in71bx6d/django-geojson/setup.py", line 15, in <module>
        open(os.path.join(here, 'CHANGES')).read(),
      File "/usr/lib/python3.5/encodings/ascii.py", line 26, in decode
        return codecs.ascii_decode(input, self.errors)[0]
    UnicodeDecodeError: 'ascii' codec can't decode byte 0xc3 in position 2183: ordinal not in range(128)
```

**Comment by shayneoAtNorwood** [comment]
woohoo! Finally!

**Comment by Gagaro** [comment]
You can thank @konstin for that one ;).

It still need a release too. Probably with the 2.0 compatibility.

**Comment by Gagaro** [comment]
It still fails with python 2.

**Comment by konstin** [comment]
I'm a bit confused by this issue vs. #59 and the closing/reopening of this issue.

I'm in favor of closing this as well as #59. The missing python 2 compatibility can imho be ignored as django 2.0 [dropped support for python 2](https://docs.djangoproject.com/en/1.11/faq/install/#what-python-version-can-i-use-with-django).

**Comment by Gagaro** [comment]
I reclosed it because it was solved by https://github.com/makinacorpus/django-geojson/commit/32cee0b2400aa14adeee5e33dd8488184c0d2ec4. I forgot to close #59.

We can't drop the support for python 2 as we still support Django 1.8 and 1.11. Of course we won't support python 2 for Django 2.0.

**Comment by Gagaro** [comment]
2.11.0 has been released

**Comment by konstin** [comment]
I'll have a look into python 2 support on friday.

**Comment by Gagaro** [comment]
It's already fixed and tested with Django 1.8 and 1.11 (https://travis-ci.org/makinacorpus/django-geojson). But thanks for the help 👍 .

**Comment by konstin** [comment]
Just checked and it works. Thanks 👍
